pages/videos/kv[kvid].vue

```diff
@@ -79,6 +102,14 @@
 		}
 	}
 
+	#recheader {
```

Comment:
avoid using IDs unless you need to. vue scopes them, but it makes fighting for specificity harder